iris_grib/_save_rules.py

```diff
@@ -942,12 +959,15 @@ def set_fixed_surfaces(cube, grib, full3d_cube=None):
         gribapi.grib_set(grib, "scaleFactorOfFirstFixedSurface", 0)
         gribapi.grib_set(grib, "scaledValueOfFirstFixedSurface", output_v)
         gribapi.grib_set(grib, "typeOfSecondFixedSurface", 255)
-        gribapi.grib_set(grib, "scaleFactorOfSecondFixedSurface", 255)
+        gribapi.grib_set(grib, "scaleFactorOfSecondFixedSurface",
+                         GRIB_MISSING_LONG)
```

Comment:
I was getting failing tests with these, so this should be GRIB_MISSING_LONG rather:
```
>>> import gribapi
>>> grib = gribapi.grib_new_from_samples("GRIB2")
>>> gribapi.grib_get(grib, "scaleFactorOfSecondFixedSurface")
2147483647
>>> gribapi.GRIB_MISSING_LONG
2147483647
```